src/main/scala/com/socrata/geospace/lib/regioncache/SpatialRegionCache.scala

```diff
@@ -108,15 +113,22 @@ class SpatialRegionCache(config: Config) extends MemoryManagingRegionCache[Spati
   private def getGeomColumnFromSoda(sodaFountain: SodaFountainClient, resourceName: String): Future[String] = {
     geomColumnCache(resourceName) {
       logger.info(s"Populating geometry column name for resource $resourceName from soda fountain..")
-      val tryColumn = SodaResponse.check(sodaFountain.schema(resourceName), Status_OK).map { jSchema =>
-        val geoColumns = jSchema.dyn.columns.!.asInstanceOf[JObject]
-                           .collect { case (k, v) if v.dyn.datatype.! == JString("multipolygon") => k }
-                           .toSeq
-        assert(geoColumns.length == 1, "There should only be one multipolygon column in region " + resourceName)
+      val tryColumn =
+        SodaResponse.check(sodaFountain.schema(resourceName), Status_OK).map { jSchema =>
+        val geoColumns = jSchema.dyn.columns.!.asInstanceOf[JObject].collect {
```

Comment:
Nit: should be indented to one indent level in from `SodaResponse.check ...` (same indent level as the `assert` line below that's also in the same block)

Nit: end curly brace of `collect` should go on next line. ie.
```scala
val geoColumns = jSchema.dyn.columns.!.asInstanceOf[JObject].collect { 
  case (k, v) if isAcceptedGeometry(v.dyn.datatype.!) => k
}.toSeq
```